submittable/project_plan.md

```diff
@@ -137,6 +137,6 @@ meetings, similar to a scrum.
 
 ## Tracking and Control Mechanisms
 
-We are using GitHub to commit changes to our documentation.
-Technologies used include Git hosted on GitHub, Discord, and Google Drive.
+We are using Git hosted on GitHub to commit changes to our documentation. 
```

Comment:
Wonderful, we shouldn't forget the underlying version control. We could also move off of GitHub if need be

Reply:
Thank you for your input!